test/nn/test_convolution.py

```diff
@@ -4055,13 +4055,13 @@ def test_conv3d_64bit_indexing(self, device):
     @skipCUDAIfRocm
     @onlyCUDA
     @largeTensorTest("20GB")
-    @largeTensorTest("80GB", "cpu")
+    @largeTensorTest("64GB", "cpu")
     def test_depthwise_conv_64bit_indexing(self, device):
-        x = torch.randn(1, 2, 32800, 32800)
-        c = nn.Conv2d(2, 2, kernel_size=3, stride=1, padding=1, groups=2)
+        x = torch.randn(1, 2, 32800, 32800, dtype=torch.half)
```

Comment:
Nit: randn has device kwarg. Any reason we don't construct directly on device to save memory and avoid a massive to copy?

Reply:
Could also parametrize dtype and do bf16 if supported :)